src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -857,7 +856,7 @@ static void WindowLoadsavePopulateList(rct_window* w, int32_t includeNewItem, co
     else
     {
         // Remove the separator at the end of the path, if present
-        safe_strcpy(_parentDirectory, absoluteDirectory, std::size(_parentDirectory));
+        safe_strcpy(_parentDirectory, absoluteDirectory.c_str(), std::size(_parentDirectory));
```

Comment:
Looks like this entire section could do with some refactoring, by using `fs::path::parent_path` for instance.

Reply:
It certainly could. I would like to address that in another PR.